src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -119,32 +130,53 @@ class PalantirDockerPlugin implements Plugin<Project> {
                 logging.captureStandardError LogLevel.ERROR
             }
 
-            if (!ext.tags.isEmpty()) {
+            Map<String, Object> tags = ext.tags.collectEntries { taskName, tagName ->
+                [generateTagTaskName(taskName), [
+                        rawTagName          : tagName,
+                        resolveTagNameAction: { -> tagName }
```

Comment:
nit: rename `rawTagName` and `resolveTagNameAction` to `tagName` and `tagTask` for clarity

Reply:
Got it.